cytosnake/utils/cytosnake_setup.py

```diff
@@ -68,13 +73,14 @@ def transport_project_files() -> None:
         src_path = pkg_path / target_dir
         target_dst = proj_path / target_dir
         if not src_path.exists():
+            print("DEBUGG:", str(src_path))
```

Comment:
```suggestion
            print(f"DEBUGG: {src_path}"))
```

Consider using the same format as the line below (or vis versa).

Reply:
oops this had to be removed thanks!